src/sentry/replays/endpoints/project_replay_recording_segment_details.py

```diff
@@ -1,9 +1,10 @@
+from __future__ import annotations
+
 from io import BytesIO
 
 import sentry_sdk
-from django.http import StreamingHttpResponse
+from django.http import HttpResponseBase
```

Comment:
```suggestion
from django.http.response import HttpResponseBase
```

Reply:
sorry tried to hack this in the github editor.. will fix